rdbms/src/main/java/io/spine/server/storage/jdbc/table/AbstractTable.java

```diff
@@ -18,74 +18,89 @@
+import io.spine.server.storage.jdbc.query.ReadQueryFactory;
+import io.spine.server.storage.jdbc.query.SelectByIdQuery;
+import io.spine.server.storage.jdbc.query.SimpleQuery;
+import io.spine.server.storage.jdbc.query.StorageIndexQuery;
+import io.spine.server.storage.jdbc.query.WriteQuery;
+import io.spine.server.storage.jdbc.query.WriteQueryFactory;
+import io.spine.server.storage.jdbc.util.DataSourceWrapper;
+import io.spine.server.storage.jdbc.util.IdColumn;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.spine3.server.storage.jdbc.Sql;
-import org.spine3.server.storage.jdbc.entity.query.DeleteAllQuery;
-import org.spine3.server.storage.jdbc.query.ContainsQuery;
-import org.spine3.server.storage.jdbc.query.DeleteRecordQuery;
-import org.spine3.server.storage.jdbc.query.QueryFactory;
-import org.spine3.server.storage.jdbc.query.SelectByIdQuery;
-import org.spine3.server.storage.jdbc.query.SimpleQuery;
-import org.spine3.server.storage.jdbc.query.WriteQuery;
-import org.spine3.server.storage.jdbc.util.DataSourceWrapper;
-import org.spine3.server.storage.jdbc.util.IdColumn;
 
 import javax.annotation.Nullable;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
 
 import static com.google.common.base.Preconditions.checkNotNull;
-import static org.spine3.server.storage.jdbc.Sql.BuildingBlock.BRACKET_CLOSE;
-import static org.spine3.server.storage.jdbc.Sql.BuildingBlock.BRACKET_OPEN;
-import static org.spine3.server.storage.jdbc.Sql.BuildingBlock.COMMA;
-import static org.spine3.server.storage.jdbc.Sql.BuildingBlock.SEMICOLON;
-import static org.spine3.server.storage.jdbc.Sql.Query.CREATE_IF_MISSING;
-import static org.spine3.server.storage.jdbc.Sql.Query.PRIMARY_KEY;
+import static io.spine.server.storage.EntityField.version;
+import static io.spine.server.storage.LifecycleFlagField.archived;
+import static io.spine.server.storage.LifecycleFlagField.deleted;
+import static io.spine.server.storage.jdbc.Sql.BuildingBlock.BRACKET_CLOSE;
+import static io.spine.server.storage.jdbc.Sql.BuildingBlock.BRACKET_OPEN;
+import static io.spine.server.storage.jdbc.Sql.BuildingBlock.COMMA;
+import static io.spine.server.storage.jdbc.Sql.BuildingBlock.SEMICOLON;
+import static io.spine.server.storage.jdbc.Sql.Query.CREATE_IF_MISSING;
+import static io.spine.server.storage.jdbc.Sql.Query.DEFAULT;
+import static io.spine.server.storage.jdbc.Sql.Query.NOT;
+import static io.spine.server.storage.jdbc.Sql.Query.NULL;
+import static io.spine.server.storage.jdbc.Sql.Query.PRIMARY_KEY;
 
 /**
  * A representation of an SQL table.
  *
- * <p>This type is responsible for storing all the information about a table including:
+ * <p>This class is responsible for storing all the information about a table including:
  * <ul>
- *     <li>Its name
- *     <li>Columns
- *     <li>Identifier and {@code PRIMARY KEY}
- *     <li>Queries to the table
+ * <li>table name;
```

Comment:
Please review this kind of auto-formatting, which destroyed the original indentation. Please restore the indentation in this and all similar cases.